storey/aggregations.py

```diff
@@ -487,6 +487,7 @@ def __init__(self, name, aggregation, explicit_windows, hidden_windows, base_tim
                 for win in self.hidden_windows.windows:
                     if win not in all_windows:
                         all_windows.append(win)
+            self._need_to_recalculate_pre_aggregates = True
```

Comment:
There is no real need to set it to True if you are calculating it on the same line, is just confusing a bit